docs/hub/models-gated.md

```diff
@@ -1,94 +1,144 @@
+#### Via the API
+
+You can automate the approval of access requests by using the API. You must pass a `token` with `write` access to the gated repository. To generate a token, go to [your user settings](https://huggingface.co/settings/tokens).
 
-| Method | URI | Description | Payload |
-| ------ | --- | ----------- | -------  |
-| `GET` | `/api/models/{repo_id}/user-access-request/pending` | Retrieve the list of pending access requests for the given model. | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `GET` | `/api/models/{repo_id}/user-access-request/accepted` | Retrieve the list of accepted access requests for the given model. | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `GET` | `/api/models/{repo_id}/user-access-request/rejected` | Retrieve the list of rejected access requests for the given model | ```headers = { "authorization" :  "Bearer $token" }``` |
-| `POST` | `/api/models/{repo_id}/user-access-request/handle` | Change the status of a given access request to `status`. | ```headers = { "authorization" :  "Bearer $token" }``` ```json = { "status": "accepted" &#124; "rejected" &#124; "pending", "user": "username" }``` |
-| `POST` | `/api/models/{repo_id}/user-access-request/grant` | Allow a specific user to access your repo. | ```headers = { "authorization" :  "Bearer $token" }``` ```json = { "user": "username" } ``` |
+| Method | URI | Description | Headers | Payload
+| ------ | --- | ----------- | -------  | -------  |
+| `GET` | `/api/models/{repo_id}/user-access-request/pending` | Retrieve the list of pending requests. | `{"authorization": "Bearer $token"}` | |
+| `GET` | `/api/models/{repo_id}/user-access-request/accepted` | Retrieve the list of accepted requests. | `{"authorization": "Bearer $token"}` | |
+| `GET` | `/api/models/{repo_id}/user-access-request/rejected` | Retrieve the list of rejected requests. | `{"authorization": "Bearer $token"}` | |
+| `POST` | `/api/models/{repo_id}/user-access-request/handle` | Change the status of a given access request to `status`. | `{"authorization": "Bearer $token"}` | `{"status": "accepted"/"rejected"/"pending", "user": "username"}` |
+| `POST` | `/api/models/{repo_id}/user-access-request/grant` | Allow a specific user to access your repo. | `{"authorization":  "Bearer $token"}` | `{"user": "username"} ` |
 
+The base URL for the HTTP endpoints above is `https://huggingface.co`.
 
-The base URL for the HTTP endpoints above is `https://huggingface.co`. The `$token` to pass as a bearer token can be generated from [your user settings](https://huggingface.co/settings/tokens). It must have `write` access to the gated repository.
+Those endpoints are not officially supported in `huggingface_hub` or `huggingface.js` yet but [this code snippet](https://github.com/huggingface/huggingface_hub/issues/1535#issuecomment-1614693412) (in Python) might help you getting started.
 
-### Notifications settings
+### Download access report
```

Comment:
This change means that old urls pointing here won't work. I think you can add something like https://github.com/huggingface/transformers/blob/eb849f6604c7dcc0e96d68f4851e52e253b9f0e5/docs/source/en/model_summary.md?plain=1#L79 to make sure old URLs still work

Reply:
@osanseviero I added backward compatible anchors in in https://github.com/huggingface/hub-docs/pull/1128/commits/79e4e99b2a581728e117fd06010a43883d64fe98. I used `<a id="...."></a>` as done [here](https://github.com/huggingface/hub-docs/blob/81c6ad79680639f5ed8ee7f1bf260993821c4c61/docs/hub/spaces-sdks-docker.md?plain=1#L25C3-L25C3).

I think using `### Encoder[[audio-encoder]]` actually changes the anchor of the title. What we want here is to be backward compatible but not necessarily change the new anchor.

(to be confirmed...)

Reply:
It works: https://moon-ci-docs.huggingface.co/docs/hub/pr_1128/en/models-gated#modifying-the-prompt

Reply:
Makes sense to me. This works, as you said: https://huggingface.co/docs/hub/spaces-sdks-docker#secret-management (but the heading is scrolled out, maybe we should place it _before_ the heading :) )

Reply:
Arf indeed :grimacing: . Updated them in https://github.com/huggingface/hub-docs/pull/1128/commits/5f3b97fa98710852d913b2076b48ba04b35ebf06